tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/TraceAnnotationsTests.cs

```diff
@@ -156,8 +156,8 @@ public async Task SubmitTraces()
 
                 // Run snapshot verification
                 var settings = VerifyHelper.GetSpanVerifierSettings();
-                await Verifier.Verify(orderedSpans, settings)
-                              .UseMethodName("_");
+                settings.UniqueForRuntime();
+                await Verifier.Verify(orderedSpans, settings);
```

Comment:
Do you know still want the `UseMethodName("_")` to keep it shorter?

Reply:
Oops, fixed (and renamed the new snapshot files). Thanks.